src/main/java/xbot/common/subsystems/autonomous/AutonomousCommandSelector.java

```diff
@@ -63,11 +60,7 @@ public Command createAutonomousStateMessageCommand(String message) {
         });
     }
 
-    public void setIsDefault(boolean bol) {
-        this.isDefault = bol;
-    }
-
-    public boolean getIsDefault() {
-        return this.isDefault;
+    public String getProgramName() {
```

Comment:
Is there a reason not to leave both so we could debug the default stuff?

One of the big reasons to go with the default approach was that we could signal if something like the Oracle being default had had any settings changed on it (which this getName() approach couldn't handle).